desktop/app-handlers/updater/manual-updater/index.js

```diff
@@ -39,8 +39,8 @@ class ManualUpdater extends Updater {
 	async ping() {
 		try {
 			const url = this.apiUrl;
-			debug( 'Checking for update. Fetching:', url );
-			debug( 'Checking for beta release:', this.beta )
+			log.info( 'Checking for update. Fetching: %s', url );
```

Comment:
Why use this to build the string? You haven't used this elsewhere. Would a template literal make more sense?

Reply:
@belcherj This is a great callout. Using `%s` and other such tokenizers is something we absolutely shouldn't have to do. I addressed this by simplifying the log formatter even further so the user doesn't have to care about what types are being passed in (via `util.format`, which essentially lets us expect the same behavior as `console.log`.) It works well! 🎉 

Fixed in d0553354fe922615cac17ee0739c0eeb35846bba